app/javascript/components/App.js

```diff
@@ -10,29 +10,25 @@ import NotFound from './pages/NotFound'
 import {
   BrowserRouter as Router,
   Route,
-  Switch
+  Routes
```

Comment:
Is Routes for the newer version of react-router-dom?

Reply:
Yeah Routes replaces Switch in react-router-doms latest version